modules/op-configuring-basic-authentication-for-git.adoc

```diff
@@ -0,0 +1,84 @@
+// This module is included in the following assembly:
+//
+// *openshift-docs/cicd/pipelines/authenticating-pipelines-using-git-secret.adoc
+
+[id="op-configuring-basic-authentication-for-git_{context}"]
+= Configuring basic authentication for Git
+
+For basic authentication, {pipelines-shortname} creates a `~/.gitconfig` file containing the credentials specified in the secret, before executing any steps in a task run or a pipeline run. When the steps execute, {pipelines-shortname} uses those credentials to retrieve the pipeline resources specified in a run.
+
+[discrete]
+.Procedure
+
+To configure a Git secret of the `basic-auth` type:
+
+. In the `secret.yaml` file, specify the username and password required to access the target Git repository.
```

Comment:
```suggestion
. In the `secret.yaml` file, specify the username and password link:https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token[GitHub personal access token] to access the target Git repository.
```

Reply:
Do we need to include steps to edit the `secret.yaml` file and then apply the changes. Is it safe to assume that the user already knows how to do this?

Reply:
I think it's better if we retain the steps to edit `secret.yaml`.